docs/data/joy/components/tabs/tabs.md

```diff
@@ -67,87 +60,62 @@ A selected `Tab` does not apply `:hover` and `:active` global variant styles.
-Keyboard navigation (e.g. arrow keys) will adapt automatically to the used orientation.
+Use the `orientation="vertical"` prop on the Tabs component to make it vertical.
+Keyboard navigation using arrow keys adapts automatically.
 
 {{"demo": "TabsVertical.js"}}
 
-### Placement
-
-To change the placement, you should provide the value of `top`, `bottom`, `left` or `right`.
+### Indicator placement
 
-This prop can be applied on the TabList component to change the `underlinePlacement` as in the example:
+Use the `underlinePlacement` prop on the Tab List component to change the Tabs' underline border placement.
 
 {{"demo": "TabsUnderlinePlacement.js"}}
 
-Or, it can be applied on the Tab component to change the `indicatorPlacement`:
+Control the selected Tab indicator independently using the `indicatorPlacement` prop.
 
 {{"demo": "TabsIndicatorPlacement.js"}}
 
-The flex direction of the Tabs component will need to be changed based on each placement.
+Depending on the underline and selected indicator placement, you may need to change the Tabs component flex-direction.
 
 {{"demo": "TabsFlexPlacement.js"}}
 
 ### Sticky
 
-For long content, you can use the `sticky="top"` prop on the TabList component to keep the tabs visible while scrolling.
-
-To stick the TabList at the bottom, use `sticky="bottom"` and render the TabList at the end of the Tabs component.
+Use the `sticky` prop to place the Tab List component at the top or bottom in a long content scenario.
 
 {{"demo": "TabsSticky.js"}}
 
 ### Tab flex
 
-Use the `tabFlex` prop on the TabList component to make the Tab elements fill the available space as shown in the example below.
+Use the `tabFlex` prop on the Tab List component to make the Tab elements fill the available space as shown in the example below.
 
-The first demo, `tabFlex={1}`, the Tab elements will fill the available space equally.
+- In the first demo, the Tab elements fill the available space using `tabFlex={1}`.
 
-The second demo, `tabFlex="auto`, the Tab elements will fill the available space equally, but the width of each Tab element will be based on the content.
+- In the second demo, the Tab elements fill the available space equally using `tabFlex="auto"`, but the width of each one of them is based on the content.
 
 {{"demo": "TabsFlex.js"}}
 
-:::success
-The value of the `tabFlex` can be any valid [CSS flex](https://developer.mozilla.org/en-US/docs/Web/CSS/flex) value.
+:::info
+The value of the `tabFlex` prop can be any [valid CSS flex value](https://developer.mozilla.org/en-US/docs/Web/CSS/flex).
 :::
 
 ### Icon
 
-Since `TabList` uses the same style as the [`List`](/joy-ui/react-list/) component, you can use the icon directly as a child or use `ListItemDecorator` with a text.
+Since the Tab List component uses the same style as the [List](/joy-ui/react-list/) component, you can use the icon directly as a child or use List Item Decorator with a text.
```

Comment:
```suggestion
Since the Tab List component uses the same style as the [List](/joy-ui/react-list/) component, you can use the icon directly as a child, or use List Item Decorator with text.
```